docs/source/kv_cache/mooncake.rst

```diff
@@ -1,11 +1,226 @@
+The system creates a distributed memory pool by aggregating memory space contributed by various client nodes, enabling efficient resource utilization across clusters.
+
+By pooling underutilized DRAM and SSD resources from multiple nodes, the system forms a unified distributed storage service that maximizes resource efficiency.
+
+.. image:: ../assets/mooncake-store-preview.png
+    :alt: MoonCake Store Architecture Diagram
+
+Key Features
+~~~~~~~~~~~~
+
+- **Distributed memory pooling**: Aggregates memory contributions from multiple client nodes into a unified storage pool
+- **High bandwidth utilization**: Supports striping and parallel I/O transfer of large objects, fully utilizing multi-NIC aggregated bandwidth
+- **RDMA optimization**: Built on Transfer Engine with support for TCP, RDMA (InfiniBand/RoCEv2/eRDMA/NVIDIA GPUDirect)
+- **Dynamic resource scaling**: Supports dynamically adding and removing nodes for elastic resource management
+
+For detailed architecture information, see the `Mooncake Store Architecture Guide <https://github.com/kvcache-ai/Mooncake/blob/main/doc/en/mooncake-store-preview.md>`_.
+
+Quick Start
+-----------
+
+Install MoonCake via pip:
+
+.. code-block:: bash
+
+    pip install mooncake-transfer-engine
+
+This package includes all necessary components:
+
+- ``mooncake_master``: Master service that manages cluster metadata and coordinates distributed storage operations
+- ``mooncake_http_metadata_server``: HTTP-based metadata server used by the underlying transfer engine for connection establishment
+- Mooncake Python bindings
+
+For production deployments or custom builds, see the `Build Instructions <https://github.com/kvcache-ai/Mooncake/blob/main/doc/en/build.md>`_.
+
+Setup and Deployment
+~~~~~~~~~~~~~~~~~~~~
+
+**Prerequisites:**
+
+- Machine with at least one GPU for vLLM inference
+- RDMA-capable network hardware and drivers (recommended) or TCP network
+- Python 3.8+ with pip
+- vLLM and LMCache installed
+
+**Step 1: Start Infrastructure Services**
+
+Start the metadata server:
+
+.. code-block:: bash
+
+    # HTTP metadata server (recommended for development)
+    mooncake_http_metadata_server
+
+Start the MoonCake master service:
+
+.. code-block:: bash
+
+    # Master service (use -v=1 for verbose logging)
+    mooncake_master
+
+Expected output:
+
+.. code-block:: text
+
+    Master service started on port 50051
+    HTTP metrics server started on port 9003
+    Master Metrics: Storage: 0.00 B / 0.00 B | Keys: 0 | ...
+
+**Step 2: Create Configuration File**
+
+Create your ``mooncake-config.yaml``:
+
+.. code-block:: yaml
+
+    chunk_size: 256
+    local_device: "cpu"
+    remote_url: "mooncakestore://127.0.0.1:50051/"
+    remote_serde: "naive"
+    pipelined_backend: False
+    local_cpu: False
+    max_local_cpu_size: 5
+
+    extra_config:
+      local_hostname: "localhost"
```

Comment:
Sorry for did not suggest here before, but #806.

It could be conflict to the extra_configs of other connectors, so, i suggest to add a prefix to each mooncakestore extra config, something like
```yaml
    extra_config:
      mooncakestore.local_hostname: "localhost"
```

And you don't need to address this in this PR.

Reply:
OK, I'll take care of it in a separate PR.